packages/block-library/src/table-of-contents/ListItem.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * WordPress dependencies
+ */
+
+export default function ListItem( props ) {
+	const { children, noWrapList = false } = props;
+	let childNodes = null;
+
+	if ( children ) {
+		childNodes = children.map( function( childNode, index ) {
+			const { content, anchor, level } = childNode.block;
+
+			const entry = anchor ? (
+				<a
+					className="blocks-table-of-contents-entry"
```

Comment:
This class name should follow the BEM-ish style used elsewhere in Gutenberg:
```suggestion
					className="wp-block-table-of-contents__entry"
```

Reply:
(The tests will have to be updated.)